microk8s/configure

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -x
+if ! [ -f ./dockercreds ]; then
+	echo "ERROR: Dockerhub credentials not found"
+	echo "Please create a file named 'dockercreds' with your dockerhub user and password"
+	echo "This is necessary in order to circumvent the API limitations in docker"
+	echo "https://docs.docker.com/docker-hub/download-rate-limit/"
+	echo "You can 'cp dockercreds.template dockercreds'"
+	echo "And adjust your credentials in the file"
+	exit 1
+fi
+
+microk8s_unit=$(juju status| sed -nr 's,(microk8s/[[:digit:]]+)\*.*,\1,p')
+juju run --unit ${microk8s_unit} -- sudo grep -q docker /var/snap/microk8s/current/args/containerd-template.toml
+if [ $? -ne 0 ]; then
+	echo "updating docker credentials on microk8s..."
+	juju scp ./dockercreds ${microk8s_unit}:
+	juju run --unit ${microk8s_unit} -- "cat /home/ubuntu/dockercreds | sudo tee -a /var/snap/microk8s/current/args/containerd-template.toml"
```

Comment:
This command is now `juju exec` in Juju 3.x. The configure script should run
with either Juju 2.9 or 3.x.

Reply:
good catch, fixed on https://github.com/canonical/stsstack-bundles/commit/0b9134098425c2295ec6e71f26ad12856b3bc630